Code/Source/Lidar/LidarRaycaster.cpp

```diff
@@ -222,27 +216,49 @@ namespace RGL
         return m_graph.IsPcPublishingEnabled();
     }
 
-    void LidarRaycaster::UpdatePublisherTimestamp(AZ::u64 timestampNanoseconds)
+    AZStd::optional<size_t> LidarRaycaster::GetRglResultsSize(
+        const PipelineGraph::RaycastResults rglResults, const ROS2::RaycastResults& results)
```

Comment:
`rglResults` is passed by copy instead of reference?

Reply:
My bad.